sbp_register.py

```diff
@@ -65,9 +91,24 @@ def on_change(self, register):
             return
 
         # Get the region
-        sbp_point_register.store(register, (self.view, self.view.window(), sel[0]))
+        sbp_point_registers.store(register, (self.view, self.view.window(), sel[0], self.view.substr(self.jove.get_encompassing_region())[0:256]))
```

Comment:
I have and continue to be too busy to look at this. Sorry about that. And github removed any other way to communicate than comments, so, sorry again!